nemo/core/optim/optimizers.py

```diff
@@ -60,16 +61,21 @@
 
         # Try importing Apex distributed Adam optimizer
         from apex.contrib.optimizers.distributed_fused_adam import DistributedFusedAdam
-        import fused_adam_cuda, distributed_adam_cuda  # Required kernels
```

Comment:
Starting in https://github.com/NVIDIA/apex/pull/1463, importing `DistributedFusedAdam` without the required kernels will cause an import error, so NeMo no longer has to make this check.